src/util/nativeStorageHelper.ts

```
@@ -0,0 +1,78 @@
+/* eslint-disable fp/no-class */
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+/**
+ *
+ */
+function handleError(func: string, param?: string): Promise<string> {
+  let message
+  if (!param) {
+    message = func
+  } else {
+    message = `${func}() requires at least ${param} as its first parameter.`
+  }
+  console.warn(message) // eslint-disable-line no-console
+  return Promise.reject(message)
```

You can throw an Error instead of returning a rejected Promise.